behandling/klage/domain/src/main/kotlin/behandling/klage/domain/VurderingerTilKlage.kt

```diff
@@ -150,16 +150,33 @@ sealed interface VurderingerTilKlage {
             data class Oppretthold(val hjemler: Klagehjemler.Utfylt) : Utfylt
         }
 
+        // Se også Omgjøringsgrunn - må se om de skal konsolideres evt fjernes fra behandlingsløpet hvis det lagres her. Blir da kun historisk
         enum class Årsak {
             FEIL_LOVANVENDELSE,
-            ULIK_SKJØNNSVURDERING,
-            SAKSBEHANDLINGSFEIL,
-            NYTT_FAKTUM,
+            NYE_OPPLYSNINGER,
```

Comment:
Dette er p.t. implementert som omgjøringsgrunn i revurdering/omgjøring av avslag. Men det er litt dumt å dobbeltlagre det så spørsmålet er om vi skal droppe å ha det her i tillegg til på de andre stedene eller heller ha det. Evt må vi migrere de eksisterende med mindre vi vil ha to kilder til sannheten her.